app/components/App.js

```diff
@@ -6,6 +6,7 @@ import Header from "./Header";
 import Home from "./Home";
 import Auction from "./Auction";
 import About from "./About";
+import AuctionList from "./AuctionList";
```

Comment:
Did you add this for development reasons?

Reply:
No, the home page will just list the current auction (like a storefront), and then `AuctionList` will have all auctions (at `/auctions`).

Reply:
Just remove this. No need for this rn